classes/Divisions.php

```diff
@@ -69,6 +69,63 @@ public function getMemberDivisionsForPolicy($policyID = null) {
         return $this->divisionsByPolicy($q);
     }
 
+    public function getMemberDivsionDetails() {
+        $args = array(':person_id' => $this->member->person_id);
+
+        $policy_divisions = array();
+
+        $q = $this->db->query(
+            "SELECT policy_id, count(division_id) as total, max(year(division_date)) as latest, min(year(division_date)) as earliest
```

Comment:
Could this query not be done in code using the results of the second query (if you added max/min year)? The second query fetches all the same data just grouped into more categories, so summing up total/min/max in that loop would also work. I don't know if this query is slow enough that that would matter, but I imagine code might be faster anyway?